test_files/module_patching.js

```diff
@@ -0,0 +1,7 @@
+console.assert(module.id === '.')
```

Comment:
just curious: why do we `console.assert` instead of using Node's built in assertion library?

Reply:
Either way works, we're just comparing outputs from running this script. I just went with what's already there.

Thanks for review. The bigger challenge now is moving node to polygott